DEVEL.md

```diff
@@ -189,3 +196,102 @@ into the `testdata` directory in this repository. Run `make test`.
 
 Assuming a successful test run, copy the required generated files to `config`.
 Build the adapter image with these changes and verify the functionality.
+
+## End-to-end walk-through
+
+The guide will walk you through the creation of the required OpenShift cluster and the deployment of
+the 3scale adapter and its' integration with a 3scale API.
+
+### Deploying OpenShift with Istio
+
+The upstream [Maistra project](https://github.com/Maistra/) contains a fork of Origin which we will use for this walk-through.
```

Comment:
Perhaps add a link to my installer gist on CentOS/RHEL? It should be ready for mass consumption now.

Reply:
Yes can do, had it in but actually removed it, thought it might be better to let Maistra take care of it but lets get it back in